test/views/messages/ReportingFrequencyViewMessages.scala

```diff
@@ -16,12 +16,18 @@
 
 package views.messages
 
-object ReportingFrequencyViewMessages {
+import models.incomeSourceDetails.TaxYear
 
-  val title = "Your reporting frequency - Manage your Self Assessment - GOV.UK"
+object ReportingFrequencyViewMessages {
+  def title(isNewContentEnabled: Boolean): String =
```

Comment:
same here in the title

instead of defs we can just create new vals for the new content and use them in the respective tests. less logic is good